src/types/onyx/ReimbursementAccount.ts

```diff
@@ -0,0 +1,37 @@
+import * as OnyxCommon from './OnyxCommon';
+
+type AchData = {
```

Comment:
I think our naming styleguide asks for uppercase shortcuts 
```suggestion
type ACHData = {
```